pymc3/distributions/timeseries.py

```diff
@@ -309,8 +311,14 @@ def _random(self, sigma, mu, size, sample_shape):
         else:
             axis = 0
         rv = stats.norm(mu, sigma)
-        data = rv.rvs(size).cumsum(axis=axis)
-        data = data - data[0]  # TODO: this should be a draw from `init`, if available
+        if len(size) == 1:
+            data = rv.rvs(size).cumsum(axis=axis)
+            data = data - data[0]  # TODO: this should be a draw from `init`, if available
```

Comment:
the comment was there before.. but do you understand what it is about?

Reply:
No, I could not make sense of it.

Reply:
a gaussian random walk's initial point comes from some distribution, and subsequent points are from the previous point with some noise. That initial distribution could be proper (like a normal) or not. If it is a distribution with no reasonable way of generating samples, then right now we just fix it to 0. 

The "TODO" means we should support drawing the initial point from the init distribution.

Reply:
Thanks a lot for the explanation.